.github/workflows/tests.yml

```diff
@@ -0,0 +1,47 @@
+name: Tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  tests:
+    name: Python ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup PostgreSQL with PostGIS
+        uses: huaxk/postgis-action@v1
+        with:
+          postgresql version: '11'
+          postgresql db: 'postgres'
+          postgresql user: 'postgres'
+
+      - name: Install PostgreSQL dependencies
+        run: |
+          sudo apt-get install -y gdal-bin
+          psql -c 'CREATE ROLE runner SUPERUSER LOGIN CREATEDB;' -U postgres -h localhost -p 5432
+          psql template1 -c "create extension citext;" -U postgres -h localhost -p 5432
+          psql template1 -c "create extension hstore;" -U postgres -h localhost -p 5432
+          psql template1 -c "create extension postgis;" -U postgres -h localhost -p 5432
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -VV
```

Comment:
Is this line for debugging purposes _oder_...?

Reply:
Right! 567b5f5